code/game/objects/items/devices/flashlight.dm

```diff
@@ -45,6 +119,23 @@
 		A.UpdateButtonIcon()
 	return 1
 
+/obj/item/flashlight/attackby(obj/item/I, mob/user)
```

Comment:
This needs to return ..() if its not a cell, or it probably makes them impossible to attack